client/my-sites/site-settings/jetpack-credentials/credentials-setup-flow/setup-footer.jsx

```diff
@@ -34,6 +38,15 @@ class SetupFooter extends Component {
 					<Gridicon icon="help" size={ 18 } className="credentials-setup-flow__footer-popover-icon" />
 					{ translate( 'Why do I need this?' ) }
 				</a>
+				<HappychatButton
+					className="credentials-setup-flow__happychat-button"
+					onClick={ this.happychatEvent }
```

Comment:
what if we made this the API for this component and instead of making the UI concern itself with calling `recordTracksEvent()` it only needs to know, "when the button is clicked, then call the click handler"

```js
mapDispatchToProps = ( {
  happychatEvent: recordTracksEvent( 'rewind_credentials…' )
} )
```

then as we update and if we update we can change `connect()` but the component need not change

```
happychatEvent: withAnalytics(
  recordTracksEvent( … ),
  doSomething()
)
```